src/coreclr/jit/emitxarch.cpp

```diff
@@ -15391,6 +15401,10 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
         case INS_vfnmsub132ss:
         case INS_vfnmsub213ss:
         case INS_vfnmsub231ss:
+        case INS_vpdpbusd:  //will be populated when the HW becomes publicly available
```

Comment:
Can we get a small issue tracking this to ensure it doesn't get lost/forgotten?

Reply:
Do you want me to open an issue now, or at the time when the code could be merged?

Reply:
Either is fine, just as long as we are tracking updating it once the official numbers become available.

Reply:
Thanks!